sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/package.scala

```diff
@@ -45,7 +46,7 @@ package object util extends Logging {
     }
   }
 
-  def fileToString(file: File, encoding: String = "UTF-8"): String = {
+  def fileToString(file: File, encoding: Charset = UTF_8): String = {
```

Comment:
Ur, this is an exceptional function signature change which is different from the rest of this PR.
Please mention this clearly in the PR description with the following code snippets.
```
- def fileToString(file: File, encoding: String = "UTF-8"): String = {
+ def fileToString(file: File, encoding: Charset = UTF_8): String = {
```

Reply:
I think the change is fine to this private method in this PR. If it's just the description in question, I can edit it.